pkg/sql/vtable/pg_catalog.go

```diff
@@ -179,6 +180,12 @@ CREATE TABLE pg_catalog.pg_class (
 	relfrozenxid INT,
 	relacl STRING[],
 	reloptions STRING[],
+	relforcerowsecurity BOOL,
+	relispartition BOOL,
+	relispopulated BOOL,
+	relreplident "char",
+	relrewrite OID,
+	relrowsecurity BOOL,
```

Comment:
do you know why `relpartbound` isn't added here? as described on https://www.postgresql.org/docs/13/catalog-pg-class.html